winter/pagination/serializer.py

```diff
@@ -0,0 +1,70 @@
+from typing import Optional
+from typing import Type
+
+from rest_framework import serializers
+from rest_framework.utils.urls import remove_query_param
+from rest_framework.utils.urls import replace_query_param
+
+from .page import Page
+
+
+class _MetaSerializer(serializers.Serializer):
+    total_count = serializers.IntegerField()
+    limit = serializers.IntegerField(source='position.limit', allow_null=True)
+    offset = serializers.IntegerField(source='position.offset', allow_null=True)
+    previous = serializers.SerializerMethodField()
+    next = serializers.SerializerMethodField()
+
+    def get_previous(self, page: Page) -> Optional[str]:
+        offset = page.position.offset or 0
+        limit = page.position.limit
+
+        if offset <= 0 or limit is None:
```

Comment:
what if offset > 0 and limit is None? shouldn't it be a offset=0, limit=None then?